pkg/util/yaml/decoder_test.go

```diff
@@ -25,42 +24,6 @@ import (
 	"testing"
 )
 
-func TestSplitYAMLDocument(t *testing.T) {
```

Comment:
If you need to move the test cases or reduce their complexity that's fine, but we should be testing all of these edge cases still.